apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -17,7 +17,7 @@ import {
   sendCodebridgeAnalyticsEvent,
   shouldShowFile,
   validateFileName,
-  validateFolderName,
+  isValidFolderName,
```

Comment:
old naming style was very inconsistent.

- `checkForDuplicateFilename` would look for duplicates, and return a message string with info on how it was a duplicate.
- `isValidFolderName` would return a boolean
- There was more elaborate validation embedded inside the `newFilePrompt` function embedded in `FileBrowser`

This is the first step towards consolidating and standardizing:
- `is*` functions will return boolean values (truthy values in the case of `isValidFileName`, due to its complexity). So we get `isValidFolderName`, `isDuplicateFolderName`, etc.
- `validate*` functions will return an error message. `validate*` functions will do more than one validation - e.g., `validateFolderName` will check for a valid folder name _and_ a duplicate folder name. This is the function that is going to be used in the prompts, instead of the anonymous inline one.